File: dataset/output-bk/MISTRAL_codestral-latest/Task107_MISTRAL_codestral-latest.cpp
#include <iostream>
#include <naming.h>

int main() {
    JNDIEnvironment *env;
    JNDIContext *ctx;
    JNDIObject *obj;

    env = jndi_create_environment();
    ctx = jndi_context_new(env);

    obj = jndi_context_lookup(ctx, "java:comp/env/jdbc/MyDB");
    if (obj != NULL) {
        std::cout << "Object found: " << obj << std::endl;
    } else {
        std::cerr << "Lookup failed" << std::endl;
    }

    jndi_context_destroy(ctx);
    jndi_environment_destroy(env);

    return 0;
}

File: dataset/output-bk/MISTRAL_codestral-latest/Task107_MISTRAL_codestral-latest.java
import javax.naming.*;
import javax.naming.directory.*;

public class JNDILookup {
    public static void main(String[] args) {
        try {
            Context ctx = new InitialContext();
            Object obj = ctx.lookup("java:comp/env/jdbc/MyDB");
            System.out.println("Object found: " + obj);
        } catch (NamingException e) {
            e.printStackTrace();
        }
    }
}